<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1324</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1324</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1324</width>
    <height>400</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>206</red>
        <green>234</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>211</red>
        <green>222</green>
        <blue>227</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>117</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>208</red>
        <green>221</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>234</red>
        <green>245</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>230</red>
        <green>244</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>206</red>
        <green>234</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>211</red>
        <green>222</green>
        <blue>227</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>117</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>208</red>
        <green>221</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>234</red>
        <green>245</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>230</red>
        <green>244</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>117</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>206</red>
        <green>234</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>211</red>
        <green>222</green>
        <blue>227</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>117</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>208</red>
        <green>221</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>117</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>103</red>
        <green>117</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>234</red>
        <green>245</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>234</red>
        <green>245</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>206</red>
        <green>234</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Berlin Sans FB</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Ingreso Datos - Simulación Control de Aviones</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>imagenes/dado.png</normaloff>imagenes/dado.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>661</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Simulación Control de Aviones</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>1261</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="title">
     <string>Ingreso Datos</string>
    </property>
    <widget class="QLineEdit" name="tiempoPrimeraLlegada">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>42</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>22</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>36</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Tiempo inicial de llegada: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="permanencia1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>71</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>196</y>
       <width>231</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Finalizacion de permanencia 5:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="permanencia2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>231</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Finalizacion de permanencia 1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Finalizacion de permanencia 3:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="permanencia5">
     <property name="geometry">
      <rect>
       <x>249</x>
       <y>193</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>20</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>156</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Finalizacion de permanencia 4:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="permanencia4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>158</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>17</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>126</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Finalizacion de permanencia 2:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="permanencia3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>129</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>15</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSimular">
     <property name="geometry">
      <rect>
       <x>1100</x>
       <y>250</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Simular!</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>831</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Distribuciones</string>
     </property>
     <widget class="QLineEdit" name="dsvStd">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>60</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="media">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>62</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>60</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="aDespegue">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>140</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="bDespegue">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>140</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Distribucion uniforme de Aterrizajes:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="aAterrizaje">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>100</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="bAterrizaje">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>100</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>57</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Distribucion normal de estadia:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Duracion:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Distribucion uniforme de Despegues:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mediaExpo">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>180</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="duracion">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>2000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Distribucion exponencial negativa de llegadas:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Media:</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>751</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>751</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>751</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>60</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Desviación:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>100</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>a:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>100</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>b:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>140</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>b:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>140</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>a:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>180</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Media:</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="capMax">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>242</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>20</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>250</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Capacidad Maxima:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>260</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar desde la fila...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtInicioFilas">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>260</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>260</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Las 400 líneas después</string>
     </property>
    </widget>
    <zorder>groupBox_2</zorder>
    <zorder>tiempoPrimeraLlegada</zorder>
    <zorder>label</zorder>
    <zorder>permanencia1</zorder>
    <zorder>label_6</zorder>
    <zorder>permanencia2</zorder>
    <zorder>label_2</zorder>
    <zorder>label_4</zorder>
    <zorder>permanencia5</zorder>
    <zorder>label_5</zorder>
    <zorder>permanencia4</zorder>
    <zorder>label_3</zorder>
    <zorder>permanencia3</zorder>
    <zorder>btnSimular</zorder>
    <zorder>capMax</zorder>
    <zorder>label_12</zorder>
    <zorder>label_31</zorder>
    <zorder>txtInicioFilas</zorder>
    <zorder>label_32</zorder>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>label_13</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tiempoPrimeraLlegada</tabstop>
  <tabstop>permanencia1</tabstop>
  <tabstop>permanencia2</tabstop>
  <tabstop>permanencia3</tabstop>
  <tabstop>permanencia4</tabstop>
  <tabstop>permanencia5</tabstop>
  <tabstop>capMax</tabstop>
  <tabstop>duracion</tabstop>
  <tabstop>media</tabstop>
  <tabstop>dsvStd</tabstop>
  <tabstop>aAterrizaje</tabstop>
  <tabstop>bAterrizaje</tabstop>
  <tabstop>aDespegue</tabstop>
  <tabstop>bDespegue</tabstop>
  <tabstop>mediaExpo</tabstop>
  <tabstop>txtInicioFilas</tabstop>
  <tabstop>btnSimular</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
